src/main/charts/jira/values.yaml

```diff
@@ -60,6 +60,12 @@ jira:
     # -- The number of consecutive failures of the Jira container readiness probe before the pod fails readiness checks
     failureThreshold: 30
 
+  accessLog:
+    # -- The path within the Jira container where the local-home volume should be mounted in order to capture access logs.
+    mountPath: "/opt/atlassian/jira/logs"
+    # -- The subdirectory within the local-home volume where access logs should be stored.
+    localHomeSubPath: "log"
```

Comment:
The same as with confluence, is the access log in `/opt/atlassian/confluence/logs/logs`?

Reply:
```
root@jira-0:/var/atlassian/application-data/jira/log# ls
access_log.2021-01-19  atlassian-jira-gc-2021-01-19_05-40-50.log  atlassian-jira.log  catalina.2021-01-19.log  host-manager.2021-01-19.log  localhost.2021-01-19.log  manager.2021-01-19.log
root@jira-0:/var/atlassian/application-data/jira/log# pwd
/var/atlassian/application-data/jira/log
```
It's going to be log directory in a local-home volume.